client/src/pages/Add.jsx

```diff
@@ -0,0 +1,73 @@
+import BoulderCard from '../components/BoulderCard';
+import AddClimbedBoulderForm from '../components/AddClimbedBoulderForm';
+import { useParams, Link } from 'react-router-dom';
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const Add = ({bouldersList}) => {
+    const { id } = useParams()
+    const currentBoulder = bouldersList.find(boulder => boulder.number === Number(id));
```

Comment:
I don't understand this: The id of each Boulder is a string, right? Why don't you just compare `boulder.id === id`, doesn't that get the job done?

Reply:
In my dummy array it was in fact a number, but I have changed it to string as this makes more sense.